xbmc/profiles/ProfilesManager.cpp

```diff
@@ -43,6 +43,7 @@
 #include "guilib/GUIWindowManager.h"
 #include "guilib/LocalizeStrings.h"
 #include "input/InputManager.h"
+#include "peripherals/Peripherals.h"
```

Comment:
This should follow the pattern of the other services that need to be removed from this class (move it down a little)